### PR TITLE
Move util/env to vendor

### PR DESCRIPTION
**Pull request by pigletfly**
**What type of PR is this?**
/kind cleanup

**What this PR does / why we need it**:

Move util/env to vendor.
Update import paths.
Update godep

Associate PR against k8s.io/utils repo: https://github.com/kubernetes/utils/pull/194

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
     None

```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs
     None

```

### CODE REVIEW

File: vendor/OWNERS [hunk 1]
```diff
@@ -1,4 +0,0 @@
-# See the OWNERS docs at https://go.k8s.io/owners
```
Comment (BenTheElder) [inline vendor/OWNERS:1]: why is this being deleted?

Comment (BenTheElder) [reply vendor/OWNERS:1]: /hold

Comment (pigletfly) [reply vendor/OWNERS]: accidentally deleted

**Review by thockin — COMMENTED** [review]
Thanks!

/lgtm
/approve

**Comment by dims** [comment]
@pigletfly this needs a rebase

**Comment by thockin** [comment]
needs rebase

**Comment by BenTheElder** [comment]
see: https://groups.google.com/g/kubernetes-dev/c/tRqW6b7gPMA for this particular rebase

**Pull request closed without merging**